cs-dca/header-definitions.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<additionalHeaders>
+    <cs-actions-java-header>
+        <firstLine>/*</firstLine>
+        <beforeEachLine> </beforeEachLine>
+        <endLine> */</endLine>
+        <afterEachLine> </afterEachLine>
+        <firstLineDetectionPattern>/*</firstLineDetectionPattern>
+        <lastLineDetectionPattern> */</lastLineDetectionPattern>
```

Comment:
Please add <emptyLineAfterHeader>true</emptyLineAfterHeader> and regenerate the license, in order to avoid Javadoc warnings.